src/test/java/org/apache/commons/imaging/formats/pnm/PgmFileInfoTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.commons.imaging.formats.pnm;
+
+import org.apache.commons.imaging.ImageReadException;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+/**
+ * Unit tests for class {@link PgmFileInfo}.
+ *
+ * @date 2017-08-01
+ * @see PgmFileInfo
+ *
+ **/
+public class PgmFileInfoTest{
+
+  @Test(expected = ImageReadException.class)
```

Comment:
I think the IDE got confused here. Better to keep consistent with number of spaces.